lib/src/domain/usecase/fetch_first_last_movie_usecase.dart

```diff
@@ -0,0 +1,12 @@
+import '../../core/usecase/i_usecase.dart';
+import '../../data/repository/my_local_repository.dart';
+import '../entity/movie.dart';
+
+class FetchFirstLastMovieUseCase extends IUseCase<Movie,dynamic> {
```

Comment:
I don't understand the name

Reply:
Yeah, Mati said the same to me, but i cannot get with other name. What the use case does is to fetch the first movie to display in the last seen section. For now it's the mock movie, maybe after changes (when a database is implemented in the next module).

Reply:
Done, good name!